app/models/user.rb

```diff
@@ -9,5 +9,7 @@ class User < ActiveRecord::Base
   include DeviseTokenAuth::Concerns::User
 
   enum role: { member: 1, journalist: 5 }
+
+  has_many :articles
```

Comment:
Is this in scope for this PR?

Reply:
We'd like to show the author when viewing single articles, so we thought that it was time to associate